Unity/Assets/Game/Audio/AudioManager.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using JCMG.Utility;
+using NaughtyAttributes;
+using UnityEngine;
+using UnityEngine.Audio;
+
+public class AudioManager : Singleton<AudioManager>
+{
+	[SerializeField] private AudioSO _audioSettings;	//TODO add [SerializeField] private float _crossfadeTime;
+	[SerializeField] private AudioMixer _masterMixer;
+	[SerializeField] private AudioMixerGroup _sfxMixer, _bgmMixer;
```

Comment:
Not required, but a good dev practice to do. The `[Required]` attribute can be used on fields to ensure they are assigned in the inspector; if left unassigned an error will show in the inspector. This can help point a dev to what still needs setup or might have broken on an object. 

I'd generally request fields to be split out as well.

 ```suggestion
	[SerializeField, Required] 
	private AudioSO _audioSettings;
	
	[SerializeField, Required] 	
	private AudioMixer _masterMixer;
	
	[SerializeField, Required] 
	private AudioMixerGroup _sfxMixer;
	
	[SerializeField, Required] 
	private AudioMixerGroup _bgmMixer;
```